scripts/build_locally.py

```diff
@@ -104,6 +105,15 @@ def run(
         # Always builds using oneMKL interfaces for the cuda target
         onemkl_interfaces = True
 
+    if target_hip is not None:
+        if target_hip == "default":
```

Comment:
We need a special handling for `python scripts/build_locally.py --target-hip=`.
Now it is equal to `python scripts/build_locally.py`, which was not intended, I guess.

Reply:
The same comment is applicable to `python scripts/build_locally.py --target=`.

Reply:
I have update this logic
Logic for `--target=` will be handled by `--target="intel"` by default 
` --target-hip=` will raise a ValueError("--target-hip requires an architecture (e.g., gfx90a)")